src/browser/provider/built-in/dedicated/chrome/cdp.ts

```diff
@@ -72,7 +120,7 @@ export async function createClient (runtimeInfo) {
     runtimeInfo.client = client;
 
     await client.Page.enable();
-    await client.Network.enable();
+    await client.Network.enable({});
```

Comment:
Is `{}` argument necessary?

Reply:
It's required by type definitions:

https://github.com/ChromeDevTools/devtools-protocol/blob/master/types/protocol-proxy-api.d.ts#L1944

As you can see `params` is not optional.

Reply:
Ok